docs/source/conf.py

```diff
@@ -76,35 +76,17 @@
 
 
 # The code below will cut version displayed in the dropdown like this:
-# tags like v0.1.0 = > 0.1
-# branch like release/0.1 => 0.1
-# main will remain main
-# if not set will fail back to main
+# By default, set to "main".
+# If it's a tag like refs/tags/v1.2.3-rc4 or refs/tags/v1.2.3, then
+# cut to v1.2
 # the version varible is used in layout.html: https://github.com/pytorch/executorch/blob/main/docs/source/_templates/layout.html#L29
-if et_version_docs:
-    # Check if starts with release/ and set the version to the number after slash
-    if et_version_docs.startswith("release/"):
-        version = et_version_docs.split("/")[-1]
-    else:
-        # Remove "v" prefix if present
-        if et_version_docs.startswith("v"):
-            et_version_docs = et_version_docs[1:]
-        # Split to major, minor, and patch
-        version_components = et_version_docs.split(".")
-
-        # Combine the major and minor version components:
-        if len(version_components) >= 2:
-            version = release = ".".join(version_components[:2])
-        else:
-            # If there are not enough components, use the full version
-            version = release = et_version_docs
-
-    html_title = " ".join((project, version, "documentation"))
-# IF ET_VERSION_DOCS not set, set version to main.
-# This can be updated to nightly and so on.
-else:
-    version = "main"
-    release = "main"
+version = release = "main"
+if et_version_docs.startswith("refs/tags/v"):
+    version = ".".join(
+        et_version_docs.split("/")[-1].split("-")[0].lstrip("v").split(".")[:2]
+    )
+    print(f"Version: {version}")
```

Comment:
It'd be helpful to do this outside of the block so we can always see the release
```suggestion
print(f"Version: {version}")
```